datacommons/populations.py

```diff
@@ -12,180 +12,97 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Data Commons Populations API Extension.
+"""Data Commons Populations wrapper functions.
 """
 
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
+import datacommons.utils as utils
+import pandas as pd
 
-from types import MethodType
-from .datacommons import Frame
-from . import utils
+import requests
 
-def PopulationsExtension(frame):
-  """ The DataCommons populations API extension.
-  Allows users to do frame.function_defined_in_this_extension
-  as if these extension functions were built in frame funcs.
-  """
-  frame.get_populations = MethodType(get_populations, frame)
-  frame.get_observations = MethodType(get_observations, frame)
-  return frame
-
-def get_populations(self,
-                    seed_col_name,
-                    new_col_name,
-                    population_type,
-                    rows=100,
-                    location_property='location',
-                    **kwargs):
-  """Create a new column with population dcid, in place.
-  The current pandas dataframe should include a column containing entity IDs
-  for geo entities. This function populates a new column with
-  population dcids corresponding to the geo entities.
-
-  Args:
-    seed_col_name: The column name that contains entity (ids) that the added
-      populations belong to.
-    new_col_name: New column name.
-    population_type: Population type like "Person".
-    max_rows: The maximum number of rows returned by the query results.
-    **kwargs: keyword properties to define the population.
-
-  Raises:
-    ValueError: when input argument is not valid.
-  """
-  if seed_col_name not in self._dataframe:
-    raise ValueError('{} is not a valid seed column.'.format(seed_col_name))
-  if new_col_name in self._dataframe:
-    raise ValueError('{} is already a column name.'.format(new_col_name))
+def get_populations(dcids, population_type, pv_map, reload=False):
+  """ Returns a list of populations associated with the given dcids.
```

Comment:
this could be optional map for populations with no pvs

Reply:
Good point, changed

Reply:
Ok for now, but left a comment in notebook about "pv_map" not being an informative var name. Can change later after discussion